dataland-frontend/src/components/pages/UploadEuTaxonomyDataForNonFinancials.vue

```diff
@@ -0,0 +1,48 @@
+<template>
```

Comment:
What is this component used for? It looks like the `UploadFormWrapper` does everything this component does.

Reply:
@MarcinStanczyk7 will look into this